modules/nasmediaAdmixerBidAdapter.md

```diff
@@ -18,12 +18,17 @@ The NasmediaAdmixer adapter doesn't support multiple sizes per ad-unit and will
     var adUnits = [
```

Comment:
Should `https://adn.admixer.co.kr:10443/prebid/ad_req` return a response given this ad unit? I tried using this in a test page and received no response from that endpoint.

https://adn.admixer.co.kr:10443/prebid/ad_req?media_key=ajj7jba3&adunit_id=20432504&req_id=259039a20289f7&referrer=http%3A%2F%2Flocalhost%3A3000%2F&os=etc&platform=pc_web&